scientific-computing-with-python/learn-encapsulation-by-drawing-a-projectile-trajectory/projectile-trajectory.py

```diff
@@ -0,0 +1,122 @@
+import math
+import re
+
+# constants
+g = 9.81
+projectile = '∙'
```

Comment:
```suggestion
G = 9.81
PROJECTILE = '∙'
```
Python convention is to use all caps for _constants_.

Reply:
This I can't bring myself to do, `g` and `G` in physics are two very different constants...

Reply:
Hmm, what about `GRAVITY_OF_EARTH` or `GRAVITATIONAL_ACCELERATION`?

Reply:
yeah, extended variable name makes sense! thank you